src/webgpu/gpu_test.ts

```diff
@@ -245,6 +247,22 @@ export class GPUTestSubcaseBatchState extends SubcaseBatchState {
       }
     }
   }
+
+  /** Skips this test case if the `langFeature` is *not* supported. */
+  requireLanguageFeatureOrSkipTestCase(langFeature: WGSLLanguageFeature) {
```

Comment:
On second thought (re: my comment from October), this should probably be on GPUTest and not GPUTestSubcaseBatchState.

The reason for this is that, unlike selectDeviceOrSkipTestCase, this doesn't have to be done before getting a GPUDevice - wgslLanguageFeatures is static.

Reply:
GPUTestBase, actually, not GPUTest. It can go next to skipIfTextureFormatNotSupported.